server/graphql/common/expenses.ts

```diff
@@ -769,21 +778,30 @@ export async function editExpense(
       );
     }
 
-    return expense.update(
-      {
-        ...cleanExpenseData,
-        lastEditedById: remoteUser.id,
-        incurredAt: expenseData.incurredAt || new Date(),
-        status: shouldUpdateStatus ? 'PENDING' : expense.status,
-        FromCollectiveId: fromCollective.id,
-        PayoutMethodId: PayoutMethodId,
-        legacyPayoutMethod: models.Expense.getLegacyPayoutMethodTypeFromPayoutMethod(payoutMethod),
-        tags: cleanExpenseData.tags,
-      },
-      { transaction: t },
-    );
+    const updatedExpenseProps = {
+      ...cleanExpenseData,
+      lastEditedById: remoteUser.id,
+      incurredAt: expenseData.incurredAt || new Date(),
+      status: shouldUpdateStatus ? 'PENDING' : expense.status,
+      FromCollectiveId: fromCollective.id,
+      PayoutMethodId: PayoutMethodId,
+      legacyPayoutMethod: models.Expense.getLegacyPayoutMethodTypeFromPayoutMethod(payoutMethod),
+      tags: cleanExpenseData.tags,
+    };
+    if (isPaidCreditCardCharge) {
+      updatedExpenseProps['data'] = { ...expense.data, missingDetails: false };
+    }
+    return expense.update(updatedExpenseProps, { transaction: t });
```

Comment:
Maybe rename, `t` to transaction and just put, 🤔 

```suggestion
    return expense.update(updatedExpenseProps, { transaction });
```

Reply:
The problem here is that "Transaction" may suggest something very different given the context.

Reply:
Ah, I see; then we could leave it I guess. Just that looks odd to have a one letter argument. 😸 😸